pkg/storage/unified/client.go

```diff
@@ -173,6 +173,18 @@ func newClient(opts options.StorageOptions,
 			return nil, err
 		}
 
+		// only enable if an overrides file path is provided
+		var quotaSvc *resource.QuotaService
```

Comment:
I don't quite understand why is quota service configured both in `unified/client.go` and `unified/sql/service.go`. I would expect it to be available only in one of client or server. What am I missing?